homeassistant/components/sensor/__init__.py

```diff
@@ -56,8 +62,17 @@
 
 DEVICE_CLASSES_SCHEMA = vol.All(vol.Lower, vol.In(DEVICE_CLASSES))
 
+# The state represents a measurement in present time
+STATE_CLASS_MEASUREMENT = "measurement"
+# The state class is unknown
+STATE_CLASS_UNKNOWN = None
+
+STATE_CLASSES = [STATE_CLASS_MEASUREMENT, STATE_CLASS_UNKNOWN]
+
+STATE_CLASSES_SCHEMA = vol.All(vol.Lower, vol.In(STATE_CLASSES))
```

Comment:
Does this handle a `None` value? Specifically I'm wondering about `vol.Lower`.

Reply:
No, that won't work. And maybe it's weird to set the `STATE_CLASS_UNKNOWN` to `None`?

Reply:
If we need an unknown class I'd name it `"unknown"`. But maybe we don't need to differentiate between that and not implemented (`None`)?

Reply:
Yeah, we don't need unknown if we have `None`.